c2corg_ui/static/partials/card_routes.html

```diff
@@ -0,0 +1,12 @@
+<a ng-click="cardCtrl.openCardPage()" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <span class="list-item-title"><span ng-if="locale.title_prefix">{{locale.title_prefix}}&nbsp;: </span>{{locale.title}}</span>
+  <span class="list-item-lang" ng-if="cardCtrl.lang != locale.lang">({{locale.lang}})</span>
+  <br>
+  <div class="list-item-info">
+    Min {{doc['elevation_min']}} m / Max {{doc['elevation_max']}} m
```

Comment:
I know but it's a very first version of the route cards with "demo" attributes. More work is needed. I am not even sure those attributes are the ones to show. Height diffs or ratings would make more sense. But that should be done in another PR.
So I don't think it is worth adding i18n here now.